docs/autoscaling.md

```diff
@@ -67,24 +67,24 @@ This chart has the metrics, observability and visualization dependencies
 
 2. then download the dependencies for the `eoapi-support` chart
 
-    ```bash
-    $ helm dependency build ./eoapi-support
-    ```
+```sh
```

Comment:
The reason most of these code blocks tabbed inside of numbers is because then if you add another number the markdown is able to auto-increment. When they are left-aligned it cannot do that. Plus my personal opinion is that they look prettier within numbered items visually aligned within under the number.

What do you think?

Reply:
yes, you are right,  I have not realized that , I will revert that the changes !!